starter_code/public/javascript/index.js

```diff
@@ -1,23 +1,23 @@
-const charactersAPI = new APIHandler("http://localhost:8000")
+const charactersAPI = new APIHandler("http://localhost:8000");
 
-$(document).ready( () => {
-  document.getElementById('fetch-all').onclick = function(){
+$(document).ready(() => {
+  document.getElementById("fetch-all").onclick = function() {
+    charactersAPI.getFullList();
+  };
 
-  }
-  
-  document.getElementById('fetch-one').onclick = function(){
-    
-  }
-  
-  document.getElementById('delete-one').onclick = function(){
-        
-  }
-  
-  document.getElementById('edit-character-form').onsubmit = function(){
-            
-  }
-  
-  document.getElementById('new-character-form').onsubmit = function(){
-                
-  }
-})
+  document.getElementById("fetch-one").onclick = function() {
+    charactersAPI.getOneRegister(id);
```

Comment:
Here you need to consume the promise:

`
 charactersAPI.getOneRegister(id).then(character => { //do stuff on the DOM})
`